services/auth.service.js

```diff
@@ -0,0 +1,68 @@
+const authUtil = require("../utils/auth.util");
+const validationUtil = require("../utils/validation.util");
+const userService = require("../services/user.service");
+require("dotenv").config();
+
+const registerUser = async (body) => {
+  const response = await userService.createUser(body);
+  if (response.status==201) {
+ 
+
+    const token = await authUtil.generateJwtToken(newUser.username);
```

Comment:
where is the `newUser` object is coming from?

Reply:
Correct parameter is **response.message.username** which contains **newUser.username** in userService.createUser layer. I resolved this bug.